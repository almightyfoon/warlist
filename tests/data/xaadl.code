c5101b_-0XhHmv4p3y3J3J4L4L2zmo313vmo313u___-0XdUhIms3y3J3J3W41414Lmo313umpmp
