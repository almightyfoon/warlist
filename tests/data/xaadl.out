https://conflictchamber.com/#c5101b_-0XhHmv4p3y3J3J4L4L2zmo313vmo313u___-0XdUhIms3y3J3J3W41414Lmo313umpmp

Retribution Army - 75 / 75 points
[Theme] Legions of Dawn

[Goreshade 4] Lord Ghyrrshyld, the Forgiven [+28]
 - Phoenix [17]
 - Sphinx [13]
Arcanist Mechanik [2]
Dawnguard Sentinel Scyir [0(4)]
Dawnguard Sentinel Scyir [4]
House Shyeel Artificer [0(5)]
House Shyeel Artificer [0(5)]
Dawnguard Destors (max) [20]
Dawnguard Sentinels (max) [18]
 - Dawnguard Sentinel Officer & Standard [4]
 - Soulless Escort (2) [2]
Dawnguard Sentinels (max) [18]
 - Dawnguard Sentinel Officer & Standard [4]
 - Soulless Escort (1) [1]

Retribution Army - 75 / 75 points
[Theme] Legions of Dawn

[Issyria 1] Issyria, Sibyl of Dawn [+29]
 - Hemera [16]
 - Imperatus [20]
Arcanist Mechanik [2]
Dawnguard Sentinel Scyir [4]
Dawnguard Sentinel Scyir [4]
Eiryss, Mage Hunter of Ios [0(7)]
Ghost Sniper [0(3)]
Ghost Sniper [3]
House Shyeel Artificer [0(5)]
Dawnguard Sentinels (max) [18]
 - Dawnguard Sentinel Officer & Standard [4]
 - Soulless Escort (1) [1]
Dawnguard Trident [16]
Dawnguard Trident [16]
